bin/utils/getconfig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#  Based on Texas Instruments Incorporated - http://www.ti.com/
+
+
+import logging
+import numpy as np
+import os
+import sys
+import yaml
+
+
+def read_file(model_dir):
+    try:
+        with open(model_dir + "param.yaml", "r") as file:
+            return yaml.safe_load(file)
+    except OSError as e:
+        logging.error("YAML file not found")
+        sys.exit(1)
+    except yaml.YAMLError as e:
+        logging.error("Error parsing file: %s" % (e))
+        sys.exit(1)
+
+
+class GetConfigYaml:
+    """
+    Class used to get parameters from a YAML file
+    Parse the parameters for a YAML file with the TI Edge AI segmentation/detection/classification models
+    Args:
+        model_dir (string): The model directory
+    """
+
+    def __init__(self, model_dir):
+        yaml_params = read_file(model_dir)
+        self.params = self.parse_params(yaml_params, model_dir)
+
+    def parse_params(self, yaml_params, model_dir):
+        def params(): return None
+
+        # Get the preprocess parameters
+        params.resize = yaml_params['preprocess']['resize']
```

Comment:
What happens if there is no 'preprocess'? We need to capture and react to these errors.